keda/Chart.yaml

```diff
@@ -8,7 +8,7 @@ kubeVersion: ">=v1.24.0-0"
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.10.2
+version: 2.10.3
```

Comment:
don't bump this version. We do it during the chart release process

Reply:
@JorTurFer i have realized that two installations will not work because the metrics server, so I will use single keda setup
`name: v1beta1.external.metrics.k8s.io` should be defined only one time (

Reply:
so, should we continue with this PR or we can close it?